app/resources/inventory.py

```diff
@@ -6,6 +6,16 @@
 from vars import game_info
 
 
+@m.user_endpoint(path=["inventory", "trade"], requires=trade_requirements)
+def trade(data: dict, user: str) -> dict:
+    element: Optional[Inventory] = wrapper.session.query(Inventory).filter_by(element_uuid=data["element_uuid"]).first()
```

Comment:
when querying for the primary key you can do the following:
```suggestion
    element: Optional[Inventory] = wrapper.session.query(Inventory).get(data["element_uuid"])
```